javascript/chronometer.js

```diff
@@ -1,26 +1,64 @@
 class Chronometer {
   constructor() {
+    this.currentTime = 0
+    this.intervalId = 0
+  
     // ... your code goes here
   }
   startClick(callback) {
+    
+    let intId = setInterval(()=>{
+      this.currentTime +=1
+    callback()
+  },1000) 
+  
+  
     // ... your code goes here
   }
   getMinutes() {
+    let minutes = 0
+    let time=this.currentTime
+    if (time>=60){
+    minutes=time/60
+    
+    }
```

Comment:
even if Jasmine is fine with it you are also expected to return a number if the time is lower than 60, so you can create an else conditional or even remove the conditional altogether and divide the time between 60.